packages/services/f-braze-adapter/vite.config.js

```diff
@@ -0,0 +1,24 @@
+// vite.config.js
+const path = require('path');
+
+module.exports = {
+    build: {
+        target: 'es2015',
```

Comment:
Just a note for the future; we probably want to add this into `f-services` as well at some point.

Reply:
Yes I will note it down